client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -21,46 +25,55 @@ type Props = ProductStoreBaseProps & {
 	siteId: number | null;
 };
 
-const Includes = () => (
-	<ul>
-		<li>Real-time backups as you edit</li>
-		<li>10GB of cloud storage</li>
-		<li>30-day activity log archive</li>
-		<li>Unlimited one-click restores from the last 30 days</li>
-		<li>WooCommerce order and table backups</li>
-		<li>Redundant cloud backups on our global network</li>
-		<li>Priority support</li>
-	</ul>
-);
+const DescriptionList: React.FC< { items?: TranslateResult[] } > = ( { items } ) => {
+	if ( ! items || ! items.length ) {
+		return null;
+	}
+
+	return (
+		<ul>
+			{ items.map( ( item ) => (
+				<li key={ `${ JSON.stringify( item ) }` }>{ item }</li>
```

Comment:
`JSON.stringify` can be expensive for JSX elements generated by `translate` and sometimes the element may not even be serializable. A less expensive solution would be to use the index from `.map` and that index is not going to change dynamically so we can use it.

```suggestion
			{ items.map( ( item, index ) => (
				<li key={ index }>{ item }</li>
```